js/scripts.js

```diff
@@ -14,7 +14,35 @@ $(document).ready(function(){
 
     ui : {
 
+      lazyLoadFooter : function(target,offset){
+
+        var $target = $(target),
+            targetLoc = $target.offset().top,
+            footerLazyLoadTriggered = false;
+
+            if(!footerLazyLoadTriggered){
+
+              $(window).scroll(function(e){
+
+                var scrollPos = $(this).scrollTop(),
+                    offSet = offset,
+                    targetScroll = targetLoc - offSet;
+
+                    if(targetScroll <= scrollPos && !footerLazyLoadTriggered){
+                      Engine.ui.footerContributors(); // run custom footer
+                      footerLazyLoadTriggered = true;
```

Comment:
Instead of this you could namespace your event, then remove it when the footer runs. For example, [:25](https://github.com/a11yproject/a11yproject.com/pull/327/files#diff-549b6b2bff82a0ad6730c0f1cd665a19R25) could be:

```javascript
$window.on('scroll.lazyLoadFooter', function(e){
```

Then here you could have

```javascript
$window.off('scroll.lazyLoadFooter');
```

This would remove the scroll event after the footer runs, cease polling on scroll.